.circleci/config.yml

```diff
@@ -30,64 +33,90 @@ commands:
           key: android=tools-v1-{{ checksum "scripts/install-android-tools.sh" }}-{{ arch }}
           paths:
             - ~/android-tools
-
-  create-launch-android-emulator:
-    description: "create and launch android emulators"
-    steps:
-      - run:
-          name: create AVD
-          command: |
-            echo "no" | avdmanager --verbose create avd --force \
-              --name "Pixel_3a_API_26" \
-              --package "system-images;android-26;google_apis_playstore;x86"
-
-      - run:
-          name: start AVD
-          command: emulator @Pixel_3a_API_26 -no-window -no-audio
-          background: true
-
-      - run:
-          name: wait for emulator
-          command: |
-            adb wait-for-device shell 'while [[ -z $(getprop dev.bootcomplete) ]]; do sleep 1; done;'
   
   npm-dependencies:
-    description: "installs all global and local npm dependencies"
+    description: "Install all global and local npm dependencies"
     steps:
       - run: npm install -g cordova
       - run: npm install -g github:apache/cordova-paramedic
```

Comment:
Should we point to a specific commit? So that it doesn't randomly break in the future again

Reply:
Probably not a bad idea 😊 I'll add that!